.circleci/releases.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 
-export LEARNINGLOCKER_VERSION="v2.6.2"
+export LEARNINGLOCKER_VERSION="v2.6.3"
```

Comment:
Should we also upgrade the default release ARG in the corresponding Dockerfile?

Reply:
It's here to have a default argument. Not specifically the last one.

Reply:
If we have to upgrade both I think there is one we can remove because they are redundant.

Reply:
Or finally remove the `ARG` default value in both Dockerfiles. WDYT?

Reply:
I totally agree

Reply:
Done